models/Product.js

```diff
@@ -1,5 +1,7 @@
+import crypto from 'crypto-random-string';
```

Comment:
`crypto` is a NodeJS built-in module, so use it instead of 3rd party package
https://nodejs.org/api/crypto.html